packages/admin-ui-docs/docs/guidelines/experimental/combobox.mdx

```diff
@@ -254,21 +411,27 @@ Features and fixes scheduled for this component:
 | Name      | Type       | Description                  | Required | Default |
 | --------- | ---------- | ---------------------------- | -------- | ------- |
 | timeoutMs | `number`   | Timeout of the deferredValue | 🚫       | -       |
-| list      | `string[]` | List of items                | 🚫       | -       |
+| list      | `ItemType[]` | List of items                | 🚫       | -       |
+| getOptionValue | `(item: ItemType) => string` | Function that transforms item shape into a string value, no need to use it on string[] lists  | 🚫       | by default it returns the item value, stringified if it is not string|
```

Comment:
```suggestion
| getOptionValue | `(item: ItemType) => string` | Function that transforms item shape into a string value, no need to use it on string[] lists  | 🚫       | by default it returns the item value, stringified if it is not a string|
```

Reply:
Not sure on this one. What do you think?

Reply:
I see "not string" like not of the type string. Like you would say "not int"